bc_obps/registration/tests/test_utils.py

```diff
@@ -320,3 +321,34 @@ def test_none_values(self):
         with self.assertRaises(ValueError) as context:
             files_have_same_hash(file1, None)
         self.assertEqual(str(context.exception), "Both files must be provided to compare hashes.")
+
+
+class TestCheckIndustryUserOperationAccess(TestCase):
+    def test_no_error_raised_when_user_has_all_permissions(self):
+        user = baker.make_recipe(
+            'utils.industry_operator_user',
+        )
+        user_operator = baker.make_recipe('utils.approved_user_operator', user=user)
+        operation = baker.make_recipe('utils.operation', operator=user_operator.operator)
+        check_industry_user_operation_access(user.user_guid, operation.id)
```

Comment:
Brianna use this instead: operation: Operation = OperationService.get_if_authorized(user_guid, operation_id)

Reply:
Not in this file, here: /home/briannacerkiewicz/cas-registration/bc_obps/service/operation_service_v2.py